build/phpstan/bootstrap.php

```diff
@@ -1,11 +1,13 @@
 <?php
 // Defined some constants and load Dolibarr env to reduce PHPStan bootstrap that fails to load a lot of things.
-//define('DOL_DOCUMENT_ROOT', __DIR__ . '/../../htdocs');
-//define('DOL_DATA_ROOT', __DIR__ . '/../../documents');
-//define('DOL_URL_ROOT', '/');
+define('DOL_DOCUMENT_ROOT', __DIR__ . '/../../htdocs');
+define('DOL_DATA_ROOT', __DIR__ . '/../../documents');
+define('DOL_URL_ROOT', '/');
+define('DOL_MAIN_URL_ROOT', '/');
+define('MAIN_DB_PREFIX', 'llx_');
 
 // Load the main.inc.php file to have functions env defined
 if (! defined("NOLOGIN")) define("NOLOGIN", '1');
 if (! defined("NOHTTPSREDIRECT")) define("NOHTTPSREDIRECT", '1');
 global $conf, $langs, $user, $db;
-include_once __DIR__ . '/../../htdocs/main.inc.php';
+// include_once __DIR__ . '/../../htdocs/main.inc.php';
```

Comment:
What is gain of removing call to main.inc.php ? This does not break knowledge of type of conf, user and db variable ?

Reply:
Test will never fail, until we remove `continue-on-error: true` from phpstan.yml but we can see errors